packages/language/src/validation/lsp-codes.ts

```diff
@@ -21,4 +21,10 @@ export const LspCodes = {
         "Could not resolve include directive. Plugin configuration is missing",
     } as SimplePLICode,
   },
+
+  UpperCase: {
+    code: "LSPUC001",
+    severity: Severity.W,
+    message: (token: any) => `Input text "${token}" must be uppercase.`,
+  } as ParametricPLICode,
```

Comment:
Suggestion: Casting will make the `Parameter<T["message"]>` type behave incorrectly.

```suggestion
    message: (token: string) => `Input text "${token}" must be uppercase.`,
  },
```